privval/signer_listener_endpoint.go

```diff
@@ -18,11 +18,15 @@ type SignerValidatorEndpointOption func(*SignerListenerEndpoint)
 type SignerListenerEndpoint struct {
 	cmn.BaseService
 
-	mtx      sync.Mutex
+	extMtx   sync.Mutex
```

Comment:
What does extMtx stand for and what does it guard?

Reply:
yes, obsolete/fixed